content/en/platform/corda/4.8/enterprise/notary/faq/notary-failover.md

```diff
@@ -13,10 +13,11 @@ title: Notary Failover
 
 # Notary Failover
```

Comment:
```suggestion
# Notary failover
```
Not sure why "Failover" is capitalized. Is "Notary Failover" a proper name of a feature? If no, change to "Notary failover".